scripts/create-a-m-o/index.js

```diff
@@ -0,0 +1,136 @@
+const outdent = require('outdent');
+const chalk = require('chalk');
+const readline = require('readline');
+
+const {
+  getAMOType,
+  prepareName,
+  createFiles,
+} = require('./functions.js');
+
+process.stdin.setEncoding('utf8');
+
+console.log(chalk.cyan(outdent`
+    Hello Dear developer, thank you for contributing with us. 😊
+
+    I will help you to create a new web component 😎.
+
+    You can choose between ATOM📗, MOLECULE📘 or ORGANISM📙.
+
+    As a general guideline, an ATOM📗 is the smallest component.
+    It won't make sense to use it on its own, but is an essential building block.
+    An ATOM📗 should not have dependencies to other elements.
+
+    An ORGANISM📙 is the finished and ready to use component. It must have at least one element as dependency.
+    As a rule of thumb, everything that has 100% width is an organism.
+
+    A MOLECULE📘 can be a finished component and can be reused somewhere else but has never 100% width.
```

Comment:
Put this before Organism

Reply:
done

Reply:
Can we shorten this a little? It's cute but not really needed.

i.e `Please choose between Atom, Molecule and Organism`

Reply:
I think is better leave it as everyone has its own Interpretation of the atomic design